AprilTagsVision/Robot.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
robot.py should be lowercase `robot.py`

Reply:
Done
https://github.com/robotpy/examples/pull/100#issuecomment-1873734226